scripts/lib/dependencies.ts

```diff
@@ -148,3 +151,19 @@ export async function getPublishedReleaseTagNames(owner: string, repo: string) {
 
   return publishedReleases.map(publishedRelease => publishedRelease.tag_name);
 }
+
+// Dependency's that adhere to the following criteria may use this function
```

Comment:
```suggestion
// Dependencies that adhere to the following criteria may use this function
```

Reply:
That was deliberate, but in general I'm confused about how to write about the `Dependency` class in plural. It communicates the idea :shrug: